libs/java/dal/db/dal-db-migration/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM maven:3.6.3-openjdk-17 AS base
```

Comment:
This image is so we can run the database migration as a docker container.  This provides us a way to run it in Openshift and locally.